Servers/UI/OJS.Servers.Ui/ClientApp/src/styles/_components.scss

```diff
@@ -25,7 +25,7 @@
   box-sizing: border-box;
   margin: 0 auto;
   max-width: 90%;
-  width: 1280px;
+  width: 100%;
```

Comment:
That is weird combination:
- `max-width: 90%`
- `width: 100%`

This should be a different exact value, i.e. pixels, depending on responsive breakpoint